### DISCUSSION TITLE
First contribution

### DISCUSSION THREAD

**Discussion by tsironis13** [spartan-ng/spartan#315] [Q&A]
I am really excited about this project and the idea of bringing the shadcn functionality into Angular world.
I am interested in offering any code contribution and more into the project and I am wondering how I can start off?

Are there any issues/bugfixes even features that I can cope with as a landing step?

**Comment by goetzrobin** [comment]
Hey there!
This could be a good one to start: https://github.com/goetzrobin/spartan/issues/223
But also adding the same examples shadcn provides on their website.
For example: https://ui.shadcn.com/examples/playground
This is a good one to start with, because you will mostly "just use" the components and it might surface bugs/enhancements that are needed to make them work.
How does that sound?

**Reply by tsironis13** [reply]
Hi Robin!
According to your comment on #223 there is something similar already implemented; the multi-toggle-group or am I missing something?
